test/gtest/miopendriver_common.hpp

```diff
@@ -39,6 +39,30 @@
 using ::testing::HasSubstr;
 using ::testing::Not;
 
+namespace miopendriver::basearg {
+namespace conv {
+static const std::string Float    = "conv";
```

Comment:
This is fine for now. In reality we should be composing it with `{conv, pool, gemm} x {f32, f16, bf16, int8, ...}`, and there should be a separate commandline option for specifying the data type. Out of scope of this PR obviously.